pages/tutorials/getting_started.md

```diff
@@ -105,7 +104,7 @@ In the output, you'll see:
 
 - A pre-command hook ran and printed some text in the logs.
 - The agent checked out the repository.
-- The agent can access different environment variables shown in the job environment.
+- The agent had accessed different environment variables shown in the job environment.
```

Comment:
"had accessed" reads a bit awkwardly to me here. Do you think we can drop "had?"

```suggestion
- The agent accessed different environment variables shown in the job environment.
```